tests/test_csc.py

```diff
@@ -242,7 +243,7 @@ async def test_invalid_track_target(self) -> None:
                         self.remote.evt_atMountState,
                         state=AtMountState.TRACKINGDISABLED,
                     )
-                    await self.fault_to_enabled()
+                    # await self.fault_to_enabled()
```

Comment:
Please delete this and the other instances below.

If you want to keep these commented-out lines, please add a TODO with a Jira ticket number and a brief explanation.

Reply:
Thanks for catching this. I kept them when modifying the CSC code in case it turned out I needed them and I forgot to delete them.